tests/contrib/psycopg/test_psycopg.py

```diff
@@ -386,6 +386,15 @@ def test_contextmanager_connection(self):
             conn.cursor().execute("""select 'blah'""")
             self.assert_structure(dict(name="postgres.query", service=service))
 
+    @snapshot()
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_DBM_PROPAGATION_MODE="full"))
+    def test_postgres_dbm_propagation(self):
+        """generates snapshot to check whether execution of SQL string sets dbm propagation tag"""
```

Comment:
where do we check that the SQL comment was added? it should be getting it right?

Reply:
In the dbapi tests we mock the cursor object and assert that the sql comment was set: https://github.com/DataDog/dd-trace-py/blob/1.x/tests/contrib/dbapi/test_dbapi.py#L54. We could do the same here but I wanted this test to:
1. make a real database query
2. assert that the query successed without raising a syntax error
3. the `_dd.dbm_trace_injected` tag was set in the snapshot test

We plan to check the sql comment added in the system test

Reply:
Is there a similar test for psycopg with the mocked cursor?

Reply:
Nope just the dbapi test. I'll add it.